prisma/migrations/20221124200902_add_kavita/migration.sql

```diff
@@ -2,4 +2,5 @@
 ALTER TABLE "Settings" ADD COLUMN     "kavitaEnabled" BOOLEAN NOT NULL DEFAULT false,
 ADD COLUMN     "kavitaHost" TEXT,
 ADD COLUMN     "kavitaPassword" TEXT,
-ADD COLUMN     "kavitaUser" TEXT;
+ADD COLUMN     "kavitaUser" TEXT,
+ADD COLUMN     "kavitaLibraries" TEXT;
```

Comment:
Can you create new migration? This will break the update if they are already on this version.